io/include/pcl/io/image_grabber.h

```diff
@@ -75,59 +75,50 @@ namespace pcl
      */
     ImageGrabberBase (const std::vector<std::string>& depth_image_files, float frames_per_second, bool repeat);
 
-    /** \brief Copy constructor.
-     * \param[in] src the Image Grabber base object to copy into this
+    /** \brief Deleted copy constructor
      */
-    ImageGrabberBase (const ImageGrabberBase &src) : impl_ ()
-    {
-      *this = src;
-    }
+    ImageGrabberBase (const ImageGrabberBase&) = delete;
 
-    /** \brief Copy operator.
-     * \param[in] src the Image Grabber base object to copy into this
+    /** \brief Deleted copy operator
      */
     ImageGrabberBase&
-    operator = (const ImageGrabberBase &src)
-    {
-      impl_ = src.impl_;
-      return (*this);
-    }
+    operator = (const ImageGrabberBase &src) = delete;
```

Comment:
Why do you need to explicitly delete these vs remove them entirely like you did in PCDGrabber?

Reply:
I'll try to simply remove them

Reply:
Done. Waiting on CI